lib/src/deploy.rs

```diff
@@ -113,6 +113,55 @@ pub(crate) fn check_bootc_label(config: &ostree_ext::oci_spec::image::ImageConfi
     }
 }
 
+/// Write container fetch progress to standard output.
+async fn handle_layer_progress_print(
```

Comment:
See https://github.com/ostreedev/ostree-rs-ext/blob/9a4743a657ffe0435018d9720c6df80a486ca0f1/lib/src/cli.rs#L566 from which this forked to add the layer count.